build.sbt

```diff
@@ -6,6 +6,7 @@ val akkaVersion = "2.5.11"
 val slickVersion = "3.2.1"
 val circeVersion = "0.11.0"
 val catsVersion = "1.6.0"
+val monocleVersion = "1.6.0-M1"
```

Comment:
M1? You are a brave man ;)

Reply:
Using the bare-minimum... will you say that the release cycle will take that long for a stable release?

Reply:
I don't how fast they are with releasing new versions but M1 sounds like the early alpha for me.

Reply:
I changed it, you're right, I don't know why but I unwillingly read M1 as RC1